docs/chapter_heap/heap.md

```diff
@@ -304,13 +304,79 @@
 === "JS"
 
     ```javascript title="heap.js"
-    // JavaScript 未提供内置 Heap 类
+    import { Heap } from 'https://esm.sh/heap-js@2.5.0';
+
+    /* 初始化堆 */
+    // 初始化小顶堆
+    const minHeap = new Heap();
```

Comment:
Could we add the `heap.js` file which is runnable? So as `heap.ts`

Reply:
`heap-js` is an npm library.

Reply:
I have already added `heap.js` and `heap.ts`.

Reply:
Thanks! Sorry that I'm not familiar with JS and TS.
Waiting for the review by @justin-tse.